internal/runtime/runtime.go

```diff
@@ -99,8 +99,7 @@ func (gr *GolangRuntime) ProcessMessage(edgexcontext *appcontext.Context, envelo
 			}
 
 		case clients.ContentTypeCBOR:
-			x := codec.CborHandle{}
-			err := codec.NewDecoderBytes([]byte(envelope.Payload), &x).Decode(&target)
+			err := cbor.Unmarshal([]byte(envelope.Payload), target)
```

Comment:
Does this need to be `&target`?

Reply:
It actually requires a pointer which is done at line 62. If you pass in &target then the output will be a map[string] interface{} which breaks the code at 129 because it is unable to get the type.

Reply:
Gotcha.  I didn't look at much beyond the diff and just wanted to doublecheck.  This makes sense, though.